projects/v2/src/app/components/doi/doi.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import {
+  MAT_BOTTOM_SHEET_DATA,
+  MatBottomSheetRef,
+} from '@angular/material/bottom-sheet';
+import { Error } from '../../models/response.model';
+import { faThemeisle } from '@fortawesome/free-brands-svg-icons';
+
+@Component({
+  selector: 'app-doi',
+  templateUrl: './doi.component.html',
+  styleUrls: ['./doi.component.scss'],
+})
+export class DoiComponent implements OnInit {
+  loading = true;
+  noId = false;
+  error: Error = { hasError: false };
+  info: any;
+  constructor(
+    @Inject(MAT_BOTTOM_SHEET_DATA) public data: any,
+    public sheetRef: MatBottomSheetRef
+  ) {}
+
+  ngOnInit(): void {
+    this.loading = false;
+    this.data = this.data.map((item) => {
```

Comment:
What is happening here?

Reply:
For each doi item the doi property starting with "doi: ". So I trimmed that part here.